CLI/Repl.cpp

```diff
@@ -444,7 +458,32 @@ int main(int argc, char** argv)
         return 0;
     }
 
-    if (argc >= 2 && strcmp(argv[1], "--compile") == 0)
+    if ((argc >= 2 && strcmp(argv[1], "--compile=text") == 0) || strcmp(argv[1], "--compile") == 0)
```

Comment:
This would be simpler if we just have `if (argc >= 2 && strncmp(argv[1], "--compile", strlen("--compile")) == 0)`, and inside the `if` we can compare the argv to determine the format. That way you get to reuse the rest of the code between two branches.